spec.bs

```diff
@@ -575,6 +629,8 @@ this extension allows a third-party to perform an authentication ceremony on
 behalf of the [=Relying Party=], and also adds transaction information to the
 signed cryptogram.
 
+Q: Where is the adding of Transaction information defined? I think we should define it in more detail in this spec?
```

Comment:
https://w3c.github.io/secure-payment-confirmation/#sctn-payment-extension-enrollment

Under 'Client extension processing (authentication)', where it modifies step 9.